app/experimenter/nimbus-ui/src/components/FormOverview/index.tsx

```diff
@@ -49,10 +52,22 @@ export const overviewFieldNames = [
   "publicDescription",
   "riskMitigationLink",
   "documentationLinks",
+  "riskBrand",
+  "riskRevenue",
+  "riskPartnerRelated",
 ] as const;
 
 type OverviewFieldName = typeof overviewFieldNames[number];
 
+const optionalBoolString = (
+  value: boolean | null | undefined,
+): string | null => {
+  if (value == null) {
+    return null;
+  }
+  return String(value);
+};
```

Comment:
This and the below `optionalStringBool` are handling converting these fields' optional boolean values to and from strings. I tried seeing if react-hook-form had any magic that could do this for us (like `setValueAs`), but didn't see anything. Open to suggestions.

Reply:
Converted to a library :)

> when is this value expected to be null vs undefined

Not sure, these are the types that were automatically generated from the GQL schema

> do we want the string "undefined" or to return literally undefined

I don't think we'd ever get there with this function as it performs a loose null check, so in this case an undefined value would return null